kuma/static/js/libs/prism/prism.css

```diff
@@ -0,0 +1,225 @@
+/* http://prismjs.com/download.html?themes=prism&languages=markup+css+clike+javascript+css-extras+rust&plugins=line-highlight+line-numbers */
+/**
+ * prism.js default theme for JavaScript, CSS and HTML
+ * Based on dabblet (http://dabblet.com)
+ * @author Lea Verou
+ */
+
+code[class*="language-"],
+pre[class*="language-"] {
+	color: black;
+	text-shadow: 0 1px white;
+	font-family: Consolas, Monaco, 'Andale Mono', 'Ubuntu Mono', monospace;
+	direction: ltr;
+	text-align: left;
+	white-space: pre;
+	word-spacing: normal;
+	word-break: normal;
```

Comment:
CSS3 word-break not supported by: Chrome (43), Safari (8,7.1), Opera (29,30), iOS Safari (8,8.1-8.4)